dev/io.openliberty.org.apache.myfaces.4.0_fat/build.gradle

```diff
@@ -24,17 +24,25 @@
 // Define G:A:V coordinates of each dependency
 dependencies {
     requiredLibs 'net.sourceforge.htmlunit:htmlunit:2.44.0'
-    requiredLibs 'org.apache.commons:commons-lang3:3.8'
     requiredLibs 'net.sourceforge.htmlunit:htmlunit-cssparser:1.6.0'
+    requiredLibs 'net.sourceforge.htmlunit:htmlunit-core-js:2.44.0'
     requiredLibs 'net.sourceforge.htmlunit:neko-htmlunit:2.44.0'
     requiredLibs 'org.apache.commons:commons-lang3:3.8'
     requiredLibs 'org.apache.httpcomponents:httpclient:4.5.7'
     requiredLibs 'org.apache.httpcomponents:httpcore:4.4.9'
     requiredLibs 'org.apache.httpcomponents:httpmime:4.5.3'
+    requiredLibs('org.eclipse.jetty:jetty-client:9.4.9.v20180320') { force = true }
+    requiredLibs('org.eclipse.jetty:jetty-http:9.4.9.v20180320') { force = true }
+    requiredLibs('org.eclipse.jetty:jetty-io:9.4.9.v20180320') { force = true }
+    requiredLibs('org.eclipse.jetty:jetty-util:9.4.9.v20180320') { force = true }
+    requiredLibs('org.eclipse.jetty.websocket:websocket-client:9.4.9.v20180320') { force = true }
+    requiredLibs('org.eclipse.jetty.websocket:websocket-api:9.4.9.v20180320') { force = true }
+    requiredLibs('org.eclipse.jetty.websocket:websocket-common:9.4.9.v20180320') { force = true }
     requiredLibs 'org.brotli:dec:0.1.2'
-    requiredLibs 'net.sourceforge.htmlunit:htmlunit-core-js:2.44.0'
     requiredLibs 'xalan:xalan:2.7.2'
+    requiredLibs 'xml-apis:xml-apis:1.4.01'
     requiredLibs project(':com.ibm.ws.org.apache.commons.io')
+    requiredLibs project(':com.ibm.ws.org.slf4j.api')
```

Comment:
are any of these updates necessary for this PR?

Reply:
They will be used in a future PR for websocket testing.  Leaving them in after discussion.